massa-sdk/src/lib.rs

```diff
@@ -172,36 +176,30 @@ impl RpcClient {
     }
 
     /// Get information on a block given its BlockId
-    pub(crate) async fn get_block(&self, block_id: BlockId) -> RpcResult<BlockInfo> {
+    pub async fn get_block(&self, block_id: BlockId) -> RpcResult<BlockInfo> {
         self.call_method("get_block", "BlockInfo", vec![block_id])
             .await
     }
 
     /// Get the block graph within the specified time interval.
     /// Optional parameters: from <time_start> (included) and to <time_end> (excluded) millisecond timestamp
-    pub(crate) async fn _get_graph_interval(
+    pub async fn _get_graph_interval(
         &self,
         time_interval: TimeInterval,
     ) -> RpcResult<Vec<BlockSummary>> {
         self.call_method("get_graph_interval", "Vec<BlockSummary>", time_interval)
             .await
     }
```

Comment:
Not sure. Maybe functions like that doesn't have to begin with a _ if it's pub ?

Reply:
Agree. Just made the modification.